.standard_todo.yml

```diff
@@ -0,0 +1,500 @@
+- spec/policies/dashboard_policy_spec.rb
+- spec/policies/manage/admin/user_policy_spec.rb
+- spec/policies/manage/facility_group_policy_spec.rb
+- spec/policies/manage/facility_policy_spec.rb
+- spec/policies/manage/organization_policy_spec.rb
+- spec/policies/manage/user/user_policy_spec.rb
+- spec/policies/overdue_list/appointment_policy_spec.rb
+- spec/policies/overdue_list/facility_policy_spec.rb
+- spec/policies/overdue_list/patient_summary_policy_spec.rb
+- spec/policies/protocol_drug_policy_spec.rb
+- spec/policies/protocol_policy_spec.rb
+- spec/presenters/user_analytics_presenter_spec.rb
+- spec/queries/cohort_analytics_query_spec.rb
+- spec/queries/district_analytics_query_spec.rb
+- spec/queries/facility_analytics_query_spec.rb
+- spec/queries/my_facilities/blood_pressure_control_query_spec.rb
+- spec/queries/my_facilities/missed_visits_query_spec.rb
+- spec/queries/my_facilities/overview_query_spec.rb
+- spec/queries/my_facilities/registrations_query_spec.rb
+- spec/queries/patient_summary_query_spec.rb
+- spec/rails_helper.rb
+- spec/requests/api/v2/blood_pressure_sync_spec.rb
+- spec/requests/api/v2/patient_sync_spec.rb
+- spec/requests/api/v2/prescription_drug_sync_spec.rb
+- spec/requests/api/v3/blood_pressure_sync_spec.rb
+- spec/requests/api/v3/blood_sugar_sync_spec.rb
+- spec/requests/api/v3/encounter_sync_spec.rb
+- spec/requests/api/v3/patient_sync_spec.rb
+- spec/requests/api/v3/prescription_drug_sync_spec.rb
+- spec/requests/api/v4/blood_sugar_sync_spec.rb
+- spec/requests/shared_examples/api/v1/shared_spec_for_sync_requests.rb
+- spec/requests/shared_examples/api/v2/shared_spec_for_sync_requests.rb
+- spec/requests/shared_examples/api/v3/shared_spec_for_sync_requests.rb
+- spec/requests/shared_examples/api/v4/shared_spec_for_sync_requests.rb
+- spec/services/appointment_notification_service_spec.rb
+- spec/services/exotel_api_service_spec.rb
+- spec/services/redis_service_spec.rb
+- spec/services/sms_notification_service_spec.rb
+- spec/spec_helper.rb
+- spec/support/rake_test_helper.rb
+- spec/support/twilio_sms_delivery_helper.rb
+- spec/swagger_helper.rb
+- spec/transformers/api/v2/appointment_transformer_spec.rb
+- spec/transformers/api/v2/medical_history_transformer_spec.rb
+- spec/transformers/api/v2/patient_transformer_spec.rb
+- spec/transformers/api/v2/prescription_drug_transformer_spec.rb
+- spec/transformers/api/v3/appointment_transformer_spec.rb
+- spec/transformers/api/v3/blood_sugar_transformer_spec.rb
+- spec/transformers/api/v3/medical_history_transformer_spec.rb
+- spec/transformers/api/v3/patient_business_identifier_transformer_spec.rb
+- spec/transformers/api/v3/patient_phone_number_transformer_spec.rb
+- spec/transformers/api/v3/patient_transformer_spec.rb
+- spec/transformers/api/v3/prescription_drug_transformer_spec.rb
+- spec/transformers/api/v3/user_transformer_spec.rb
+- spec/transformers/api/v4/blood_sugar_transformer_spec.rb
+- spec/transformers/api/v4/user_transformer_spec.rb
```

Comment:
HAHA _adds standardrb, adds entire repo to ignorelist_

JK, this is a great way to chip away at refactors.